scripts/create_binary_segment.py

```diff
@@ -0,0 +1,104 @@
+import os
+import sys
+from glob import glob
+dynamic_path = os.path.abspath(__file__+"/../../")
+# print(dynamic_path)
+sys.path.append(dynamic_path)
+from typing import List
+import numpy as np
+import cv2
+import argparse
+
+
+def sort_file_list(file_list: List[str]) -> List[str]:
+    '''
+    Sort a list of files and return the sorted list.
+    :param file_list: list of files to sort
+    '''
+    new_list = sorted(file_list, key=lambda x: x.split('/')[-1].split('.')[0])
+    return new_list
+
+
+def color_to_binary(src_path: str, dest_folder: str, new_name: str) -> None:
+    '''
+    Convert color image to binary image
+    :param src_path: source image file path
+    :param dest_folder: destination folder path
+    :param new_name: destination image file name
+    '''
+    # Copy and Rename the copied file
+    new_path = os.path.join(dest_folder, new_name)
+    color_segment = cv2.imread(src_path)  # bgr
+    # Define color range
+    lower_bound = np.array([0, 80, 0])
+    upper_bound = np.array([0, 255, 0])
+    binary_segment = cv2.inRange(color_segment, lower_bound, upper_bound)
+    green_mask = np.zeros_like(color_segment, dtype=np.uint8)
+    green_mask[binary_segment == 255] = [0, 255, 0]
+    # print('test')
+    # cv2.imwrite(new_path, binary_segment)
+    cv2.imwrite(new_path, green_mask)
+
+
+class BinarySegment:
+    def __init__(self, src_folder: str):
+        '''
+        :param src_folders: source folders path
+        '''
+        self.src_folder = src_folder
+        self.subfolder_name = 'segmented'
+        self.outfolder_name= 'binary_segmented'
+
+        assert os.path.exists(os.path.join(self.src_folder, self.subfolder_name)), 'Segmented Image folder does not exist!'
```

Comment:
Let's improve this error message to something like:

`Segmented Image folder does not exist! Does your input path include the split that you want to process? For example ./AMBF_DATASETS_NEW/train`